integration-cli/docker_cli_exec_test.go

```diff
@@ -308,7 +308,7 @@ func (s *DockerSuite) TestInspectExecID(c *check.C) {
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
 	id := strings.TrimSuffix(out, "\n")
 
-	out, err := inspectField(id, "ExecIDs")
+	out, err := inspectField(id, ".ExecIDs")
```

Comment:
You don't need dot https://github.com/docker/docker/blob/master/integration-cli/docker_utils.go#L923